### PR TITLE
ADEN-1878 Use post scribe for 71Media

### PR DESCRIPTION
**Pull request by nandy-andy**
We are using GhostWriter currently on german wikias where we should use PostScribe instead. Changes below may not be too elegant but are fixing the issue without touching code in many places.

### CODE REVIEW

**Comment by nandy-andy** [comment]
Unit tests are failing because of wrong DB configuration :roller_coaster: Not connected to the changes above :japanese_ogre:

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php [hunk 1]
```diff
@@ -104,6 +109,9 @@ public static function onWikiaSkinTopScripts( &$vars, &$scripts ) {
 		// GA vars
 		$vars['wgGaHasAds'] = isset($adContext['opts']['showAds']);
 
+		// 71Media
+		$vars['wgUsePostScribe'] = $wg->UsePostScribe;
+
```
Comment (gabrys) [inline extensions/wikia/AdEngine/AdEngine2Hooks.class.php]: Let's use wgUsePostScribe and wgTitle

File: extensions/wikia/AdEngine/AdEngine2Hooks.class.php [hunk 2]
```diff
@@ -45,6 +45,11 @@ public static function onAfterInitialize( $title, $article, $output, $user, WebR
 		$wgEnableKruxTargeting = !$wgAdEngineDisableLateQueue && !$wgNoExternals && $wgEnableKruxTargeting;
 		$wgEnableKruxOnMobile = $request->getBool( 'enablekrux', $wgEnableKruxOnMobile && !$wgNoExternals );
 
+		// use post scribe with 71Media - check scriptwriter.js:35
```
Comment (gabrys) [inline extensions/wikia/AdEngine/AdEngine2Hooks.class.php]: PostScribe

**Comment by gabrys** [comment]
lgtm

**Pull request merged**